server/src/concepts/user/repositories/commands.js

```diff
@@ -0,0 +1,63 @@
+import User from "../model/User";
+
+const createUser = (data) => {
+  const newUser = new User({
+    ...data,
+  });
+
+  try {
+    newUser.save();
+    return newUser;
+  } catch (err) {
+    console.log(err);
+    return "error";
+  }
+};
+
+const updateUser = () => {
+  User.updateOne({}, function (err, result) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log("Updated: ", result);
+    }
+  });
+};
+
+const updateAllUsers = () => {
+  User.updateMany({}, function (err, result) {
```

Comment:
the same as above, invalid syntax about this method https://mongoosejs.com/docs/api.html#model_Model.updateMany
But I consider, will we need this method? Will we want to update allUsers? I think no, but maybe there is something